CHANGES.md

```diff
@@ -20,12 +20,14 @@
   - Modify GC to work with new chunked suffix. See `examples/gc.ml` for a
     demonstration of how it works with the new `split` function. (#2126,
     @metanivek)
+  - Add GC generation in GC stats (#2129, @Ngoguey42)
 
 ### Fixed
 
 - **irmin-pack**
   - Fix read-only opening flags of mapping for read-only opening of stores that
     do not have read-write rights on the file system (#2121, @Ngoguey42)
+  - Correctly report stats at the end of GC (#2129, @Ngoguey42)
```

Comment:
Do we need changelog entries since the new stats haven't been released yet?

Reply:
Good point. That's not needed